class.jetpack.php

```diff
@@ -2782,6 +2794,9 @@ public static function do_version_bump( $version, $old_version ) {
 			// Setting up jetpack manage
 			Jetpack::activate_manage();
 		}
+
+		Jetpack::activate_markdown();
```

Comment:
If the user does re-deactivate Markdown somehow or other -- wouldn't this break that by re-activating it on every single version bump forever?